Sekoia.io/sekoiaio/operation_center/update_alert_status.py

```diff
@@ -0,0 +1,45 @@
+from posixpath import join as urljoin
+
+from sekoia_automation.action import Action
+import requests
+from tenacity import retry, wait_exponential, stop_after_attempt
+
+
+STATUS_UUIDS = {
+    "PENDING": "2efc4930-1442-4abb-acf2-58ba219a4fd0",
+    "ACKNOWLEDGE": "8f206505-af6d-433e-93f4-775d46dc7d0f",
+    "ONGOING": "1f2f88d5-ff5b-48bf-bbbc-00c2fff82d9f",
+    "REJECT": "4f68da89-38e0-4703-a6ab-652f02bdf24e",
+    "CLOSE": "1738b1c1-767d-489e-bada-19176621a007",
+}
+
+
+class UpdateAlertStatus(Action):
+
+    def url(self, alert_uuid: str) -> str:
+        return urljoin(self.module.configuration["base_url"], f"api/v1/sic/alerts/{alert_uuid}/workflow")
+
+    @property
+    def headers(self) -> dict:
+        api_key = self.module.configuration["api_key"]
+        return {"Authorization": f"Bearer {api_key}"}
+
+    @retry(
+        reraise=True,
+        wait=wait_exponential(max=10),
+        stop=stop_after_attempt(5),
```

Comment:
Increase time to retry, we already had errors during many minutes. Also maybe we shouln't retry 4xx errors.


```suggestion
        wait=wait_exponential(max=300),
        stop=stop_after_attempt(10),
```

Reply:
Thank you, I'll change these